### ISSUE TITLE
Hello!

### ISSUE THREAD

**Issue by jeff-hykin** [aPinix/indent-jump-vscode#1]
I like your icon, and your idea! If this extension was released first I probably wouldn't have made mine haha
But I'd figure I'd mention it since you might like it. I've been using it for a few years now

https://marketplace.visualstudio.com/items?itemName=jeff-hykin.mario

**Comment by aPinix** [comment]
Hi there @jeff-hykin,

Wow, I didn't know your plugin existed. It looks fantastic and is packed with features.
I believe that I would be the one who would not have done another plugin eh eh.
Nevertheless, It seems mine works a little bit differently though.

Thanks for pointing out your awesome plugin mate.

Cheers, I hope you the best

**Comment by jeff-hykin** [comment]
> Wow, I didn't know your plugin existed. It looks fantastic and is packed with features.

Thanks 👍 its funny I did it out of procrastination, but it ended up being helpful.

> Nevertheless, It seems mine works a little bit differently though.

Yeah, there's a ton of hand-picked design that  **really** change the feel/functionality. I was surprised how many options there were.

There's two big ones I feel are worth mentioning.

- In the 2nd iteraiton, I made a tough decision to have the cursor basically "overshoot" a block instead of jumping to the bottom/top of the current block. It makes long distance movement faster, but comes at the cost of needing to do down+up to get to the bottom of the current block.

- The other serious change was enabling down/up stay at the current indent level, even if it has to jump WAY down to an entirely different hierarchy. It makes it easy to undo an overshoot, and helps inside nested lists, but can be really unintuitive.

Both of those could be easily be undesirable, I've considered making it a setting.

Worst of all, despite this and all my shortcuts, I still find some movement fastest with the mouse 😤 So there's still a lot of improvements to be had. 

> Thanks for pointing out your awesome plugin mate. Cheers, I hope you the best

You as well!
